pkg/util/timeutil/timeutil.go

```diff
@@ -2,6 +2,34 @@ package timeutil
 
 import "time"
 
+// ParseISO8601 parses string as ISO8601 date. Note that the format allows different timezones, also time is optional,
+// so we have to list all possible formats
 func ParseISO8601(dateTime string) (time.Time, error) {
-	return time.Parse(dateTime, "2006-01-02T15:04:05-0700")
+	parse, err := time.Parse("2006-01-02T15:04:05-07", dateTime)
```

Comment:
nope, the docs say about "Timestamp in ISO 8601" and in practice this means different variations including the time zones

Reply:
I still think Atlas returns UTC only, but thinking about it - this might be useful in the future if we decide to change/validate user input somehow since user input can contain a timezone shift.

Reply:
yes, Atlas returns UTC only but accepts any valid ISO 8601 date - so I tried to cover all possible formats :)